pints/toy/_rosenbrock.py

```diff
@@ -18,13 +18,13 @@ class RosenbrockError(pints.ErrorMeasure):
     https://en.wikipedia.org/wiki/Rosenbrock_function):
 
     .. math::
-        f(x,y) = (a - x)^2 + b(y - x^2)^2
+        f(x,y) = (1 - x)^2 + 100(y - x^2)^2
 
     *Extends:* :class:`pints.ErrorMeasure`.
     """
-    def __init__(self, a=1, b=100):
```

Comment:
@MichaelClerx It was because this function is only ever used with those parameter values and, if different values are chosen, it is possible to get a function that behaves in a way that is not conducive to MCMC (and can result in improper densities that cannot be normalised). I think it's better to reduce functionality and take away these potential issues from eventual users.

Reply:
After all, a user wanting to try a different type of Rosenbrockish function can always simply code this up themselves. We want this function to be useful not a panacea on all good and bad test Rosenbrock-type functions.